packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -0,0 +1,223 @@
+---
+title: OIN submission requirements
+meta:
+  - name: description
+    content: Use this guide to prepare all the requirements prior to submitting your integration to Okta for publication in the Okta Integration Network.
+layout: Guides
+---
+
+This guide provides you with a list of requirements you need to prepare prior to submitting your integration for publication in the [Okta Integration Network (OIN)](https://www.okta.com/integrations/).
+
+---
+
+**Learning outcome**
+
+* Understand the requirements necessary to submit your integration using the [OIN Manager](https://oinmanager.okta.com).
+
+**What you need**
+
+* a built and tested OIN integration that's ready for Okta verification
+
+---
+
+## Overview
+
+Prior to using the [OIN Manager](https://oinmanager.okta.com)&nbsp;to submit your OIN integration, you need to review the following guidelines to prepare for the artifacts requested during the submission process:
+
+* [Logo guidelines](#logo-guidelines)
+* [App description guidelines](#app-description-guidelines)
+* [Use case guidelines](#use-case-guidelines)
+* [Customer support contact guidelines](#customer-support-contact-guidelines)
+* [Test account guidelines](#test-account-guidelines)
+* [Customer configuration document guidelines](#customer-configuration-document-guidelines)
+
+Before you submit your integration, ensure that your integration doesn't use unsupported OIN Okta features. See [OIN limitations](#oin-limitations).
+
+## Logo guidelines
+
+The app logo that you submit to the OIN Manager in the **General Settings** > **App icon** field must conform to the following guidelines:
+
+* The logo file size must be less than one MB.
+* The logo dimension must be square (1:1 aspect ratio).
+
+   > **Note:** OIN app icons are placed in a 200 x 200 pixel grid.
+
+* Submit an icon rather than a wordmark (a logo that includes the company or product name).
```

Comment:
I need to use active language. What about adjusting your suggestion to this?
`- The logo needs to be an icon rather than a wordmark (a graphic that includes the company or product name).`

Reply:
Sounds good.

Reply:
Sorry, after reviewing all the comments on this section (it's been revised so many times, I lost count. LOL), I'm going to change this list to imperatives. Thanks for your thoughtful comments 👍